python/cuda_parallel/README.md

```diff
@@ -7,7 +7,7 @@ Please visit the documentation here: https://nvidia.github.io/cccl/python.html.
 ## Local development
 
 ```bash
-pip3 install -e ../cuda_cccl
-pip3 install -e .[test]
+pip3 install ../cuda_cccl
+pip3 install .[test] -v
```

Comment:
Is the `-v` intentional?

Reply:
Yeah I think it's useful to see what's being built - no strong feelings here, just something I use all the time